src/main/g8/packager/README.md

```diff
@@ -6,13 +6,14 @@ the `application.conf` file from the `backend` module.
 
 ## How to create a package?
 
-You can create pretty `zip` package with your application using the following command: 
-`sbt packager/universal:packageBin`. Your package will be created in the `packager/target/universal` directory.
-Call a starting script from the `bin` directory to start the application.
+You can create a universal production package with your application using the following command: 
+`sbt packager/universal:packageBin`. 
+Your package will be bundled in a ZIP archive in the `packager/target/universal` directory.
+Call the `bin/$name` starting script to start the application. Please not that resource access is dependent on the working directory, so you should run it exactly as above, from directory containing `lib` and `UdashStatics` directories - otherwise the webserver won't find the assets (e.g. `index.html`).
```

Comment:
```suggestion
Call the `bin/$name` starting script to start the application. Please note that resource access is dependent on the working directory, so you should run it exactly as above, from the directory containing `lib` and `UdashStatics` directories - otherwise the webserver won't find the assets (e.g. `index.html`).
```